primitives/src/locktime/absolute.rs

```diff
@@ -91,12 +91,30 @@ impl LockTime {
     pub const SIZE: usize = 4; // Serialized length of a u32.
 
     /// Constructs a new `LockTime` from a prefixed hex string.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use bitcoin_primitives::absolute::LockTime;
+    /// let hex_str = "0x2ba7b4";
```

Comment:
Same comment as elsewhere. Perhaps add a code comment describing the value and/or use a arbitrary but somewhat meaningful value.

Reply:
Done